phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -4354,6 +4360,18 @@ private PhoenixConnection upgradeSystemTask(
         return metaConnection;
     }
 
+    private PhoenixConnection upgradeSystemTransform(
+            PhoenixConnection metaConnection,
+            Map<String, String> systemTableToSnapshotMap)
+            throws SQLException {
+        try (Statement statement = metaConnection.createStatement()) {
+            statement.executeUpdate(getTransformDDL());
+        } catch (TableAlreadyExistsException ignored) {
+
```

Comment:
nit: better to add logging here so that we know the creation of the table is failed